### ISSUE TITLE
Remove alpha tag from Leading Story Promo

### ISSUE THREAD

**Issue by DenisHdz** [bbc/simorgh#4498]
**Is your feature request related to a problem? Please describe.**
The Leading Story Promo component is currently an alpha component.

This component should be amended and tested according to the a11y requirements to remove the tag.

**Describe the solution you'd like**

**Describe alternatives you've considered**
 Remove alpha tag after successful a11y swarm on the component.

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**
